mpo.rga.Dayon.yml

```diff
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home:rw
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - tar -xf dayon.tgz
+      - install dayon/dayon.jar /app
+      - install dayon/dayon_*.sh /app
+      - install dayon.sh /app
+      - install dayon.browser /app
+      - install -d /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 dayon/dayon.png /app/share/icons/hicolor/128x128/apps/mpo.rga.Dayon.png
+      - install -Dm644 dayon.assisted.desktop /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --remove-key=Exec /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/dayon_assisted.sh' /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --set-icon=mpo.rga.Dayon /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - install -Dm644 dayon.assistant.desktop /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --remove-key=Exec /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/dayon_assistant.sh' /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --set-icon=mpo.rga.Dayon /app/share/applications/mpo.rga.Dayon.assistant.desktop
+    sources:
+      - type: file
+        dest-filename: dayon.tgz
+        only-arches:
+          - x86_64
+        url: https://github.com/RetGal/Dayon/releases/download/v11.0.5/dayon.11.0.5.tgz
+        md5: b66651fe43bff87ffbf8bc794f1a665d
+      - type: file
+        dest-filename: dayon.sh
+        url: https://raw.githubusercontent.com/RetGal/Dayon/master/flatpak/dayon.sh
```

Comment:
you shoud NOT checkout files from a branch - a specific commit or tag is needed. (also below)

Reply:
also this file is in the archive.

Reply:
Indeed, but I had to alter it to make it compatible with flatpak (which supports no java-alternatives )